assets/javascripts/PerformancePage.js

```diff
@@ -0,0 +1,93 @@
+function addDatasetChart() {
+  const sql = `
+    SELECT
+      strftime('%Y-%m', entry_date) AS month_year,
+      -- Extract Year-Month from entry_date
+      strftime('%Y', entry_date) AS year,
+      -- Extract Year
+      CASE
+        strftime('%m', entry_date)
+        WHEN '01' THEN 'January'
+        WHEN '02' THEN 'February'
+        WHEN '03' THEN 'March'
+        WHEN '04' THEN 'April'
+        WHEN '05' THEN 'May'
+        WHEN '06' THEN 'June'
+        WHEN '07' THEN 'July'
+        WHEN '08' THEN 'August'
+        WHEN '09' THEN 'September'
+        WHEN '10' THEN 'October'
+        WHEN '11' THEN 'November'
+        WHEN '12' THEN 'December'
+      END AS month,
+      -- Convert month number to month name
+      COUNT(dataset) AS dataset_count,
+      SUM(COUNT(dataset)) OVER (
+        ORDER BY
+          strftime('%Y-%m', entry_date)
+      ) AS cumulative_count
+    FROM
+      dataset
+    WHERE
+      collection != ''
+    GROUP BY
+      month_year
+    ORDER BY
+      month_year;
+  `
+
+  fetch(`https://datasette.planning.data.gov.uk/digital-land.json?sql=${encodeURIComponent(sql)}`)
```

Comment:
@eveleighoj - this is throwing a CORs error, is there a way round this problem? Do I need to create a route to proxy the request?